lib/logmatcher.h

```diff
@@ -66,6 +66,7 @@ struct _LogMatcher
   gint ref_cnt;
   gint flags;
   gchar *pattern;
+  gchar *prefix;
```

Comment:
This should go in the PCRE specific matcher class, that is defined in logmatcher.c and is called LogMatcherPcreRe

Reply:
Thanks! But I have a question. If I define the `prefix` in the `LogMatcherPcreRe` class, I can't set the prefix in the `regexp-parser.c` because the definition of `LogMatcherPcreRe` is in the `logmatcher.c`. I can think of two ways to solve this problem:

1. move the definition of `LogMatcherPcreRe` from `logmatcher.c` to `logmatcher.h` (but may be I should also move the definitions of `LogMatcherGlob` and `LogMatcherString` to  `logmatcher.h` for consistency)
2. declare a new function called `log_matcher_pcre_set_prefix` in `logmatcher.h`, which can set the prefix of a LogMatcherPcreRe object